picard/ui/searchdialog/__init__.py

```diff
@@ -133,6 +133,28 @@ def trigger_search_action(self):
 Retry = namedtuple("Retry", ["function", "query"])
 
 
+BY_NUMBER, BY_DURATION = range(2)
+
+
+class SortableTableWidgetItem(QtWidgets.QTableWidgetItem):
+
+    def __init__(self, sort_key):
+        super().__init__()
+        self.sort_key = sort_key
+
+    def __lt__(self, other):
```

Comment:
Is this everything it needs to generate a sort order? I thought you needed atleast `__eq__` and functools.total_ordering

Reply:
Yes, it is everything needed.
See https://docs.python.org/3/library/stdtypes.html#list.sort for example:
"This method sorts the list in place, using only < comparisons between items."